index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://kit.fontawesome.com/5d13e94c8b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="nav">
+        <h1>Kayonga Chiteta</h1>   
+        <i class="fas fa-bars two"></i>
+    </nav>
```

Comment:
- [ ] @Kayonga99, you did well but I think you can still improve your code by using semantic elements. I suggest you wrap the whole element in a `header` tag and use the `nav` to wrap the `i` tag at line 14. Please, make sure that your name is also wrapped in an anchor (`a`) element. By this, your name will be clickable and navigates to wherever you may want to take the users.
**Example**
```
<header>
  <a href="#"><h1>Kayonga Chiteta</h1></a>
  <nav><i class="fas fa-bars two"></i></nav>
</header>
```